OrchardCore.Commerce.Tests/CurrencyTests.cs

```diff
@@ -18,7 +18,7 @@ public static CurrencyTheoryData TestData
                 { CanadianDollar, 1234.56m, "$1,234.56" },
                 { SwissFranc, 1234.56m, "CHF 1’234.56" },
                 { ChineseYuan, 1234.56m, "¥1,234.56" },
-                { new Currency("My FOO", "My FOO", "f", "FOO"), 1234.56m, "(FOO) 1,234.56" }
+                { new Currency("My FOO", "My FOO", "f", "FOO"), 1234.56m, $"(FOO) {1234.56m.ToString("N")}" }
```

Comment:
I don't think this is appropriate. Here, see he sets an array of fixed values defined with a "Country" or if you prefer "Zone" and then he creates a Currency, which should take any string and assert that it is displaying fine in each "Zone". So here you are forcing a culture currency format on the string to make sure it is compliant with your system but it should not test that, but if it's going to work with ANY culture.

Reply:
So if you enter "1,234.56" in SwissFranc it should display "CHF 1’234.56".
If your culture doesn't use a comma then it should pass too. If it doesn't then there's an issue in the Currency string parser.